src/views/Project/index.jsx

```diff
@@ -22,15 +25,58 @@ const tagReposMapping = repositories =>
   }, {});
 
 @hot(module)
-@graphql(issuesQuery, {
-  skip: props => !projects[props.match.params.project].repositories,
-  options: () => ({
-    fetchPolicy: 'network-only',
-    variables: {
-      searchQuery: '',
-    },
+@compose(
+  graphql(issuesQuery, {
+    skip: ({
+      match: {
+        params: { project },
+      },
+    }) => !projects[project].repositories,
+    name: 'github',
+    options: () => ({
+      fetchPolicy: 'network-only',
+      variables: {
+        searchQuery: '',
+      },
+      context: {
+        client: 'github',
+      },
+    }),
   }),
-})
+  graphql(bugsQuery, {
+    skip: ({
+      match: {
+        params: { project },
+      },
+    }) => !projects[project].products,
+    name: 'bugzilla',
+    options: ({
+      match: {
+        params: { project },
+      },
+    }) => ({
+      variables: {
+        search: {
+          keywords: ['good-first-bug'],
```

Comment:
`keywords` and `statuses` can be extracted to the top and reused here and inside `fetchBugzilla` via the spread operator:

```jsx
const bugzillaSearchOptions = {
  keywords: ['good-first-bug'],
  statuses: ['NEW', 'UNCONFIRMED', 'ASSIGNED', 'REOPENED'], 
};
```

Reply:
Also, let's create a constants file in the utils directory and have `good-first-bug`, `NEW`, `UNCONFIRMED`, `ASSIGNED`, `REOPENED` defined there.

```jsx
export const GOOD_FIRST_BUG = 'good-first-bug';
export const BUGZILLA_STATUSES = {
  NEW: 'NEW',
  ...
}
```